### PR TITLE
Added lazylibrarian application sync

### PR DESCRIPTION
**Pull request by philborman**
#### Database Migration
NO

#### Description
Adds lazylibrarian support to Prowlarr, list/add/delete/change providers

#### Todos
- [ ] Tests
- [ ] Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [ ] [Wiki Updates](https://wiki.servarr.com)

Ref #86

### CODE REVIEW

File: LazyLibrarianField.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+namespace NzbDrone.Core.Applications.LazyLibrarian
```
Comment (Qstick) [inline LazyLibrarianField.cs:1]: Purge this file, not used

**Review by Qstick — CHANGES_REQUESTED** [review]

**Comment by philborman** [comment]
Sorry, just cloned the mylar code and tweaked it a bit. Uploaded changes.

**Comment by bakerboy448** [comment]
There's still many outstanding requested changes pending.

Refer to the mylar3 cleanup commit for some more info https://github.com/Prowlarr/Prowlarr/commit/c72222a696752f837bcbb6001f5aaa0c4c9fdb77

**Comment by philborman** [comment]
All the changes are made here, looks like the file upload doesn't overwrite the earlier uploaded files.
I have attached new copies. If this doesn't work, can you delete the merge request and I will open a new one. Thanks,
[LazyLibrarian.zip](https://github.com/Prowlarr/Prowlarr/files/7071088/LazyLibrarian.zip)

**Comment by bakerboy448** [comment]
just make a commit with the changes for the files to the branch on your fork

https://docs.github.com/en/github/collaborating-with-pull-requests/getting-started/about-collaborative-development-models

**Pull request closed without merging**